addons/account/models/account_analytic_line.py

```diff
@@ -87,3 +87,42 @@ def view_header_get(self, view_id, view_type):
                 account=self.env['account.analytic.account'].browse(self.env.context['account_id']).name
             )
         return super().view_header_get(view_id, view_type)
+    
+    def _compute_key(self, analytic_line):
```

Comment:
`_compute_*` is for the functions computing the values of a field
```suggestion
    def _get_distribution_key(self, analytic_line):
```